frontend/server/src/Controllers/SubmissionFeedback.php

```diff
@@ -0,0 +1,166 @@
+                $feedback,
+                1,
+                200
+            )
+        );
+
+        $courseAlias = $r->ensureString(
+            'course_alias',
+            fn (string $alias) => \OmegaUp\Validators::alias($alias)
+        );
+
+        $course = \OmegaUp\DAO\Courses::getByAlias($courseAlias);
+        if (is_null($course)) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'courseNotFound'
+            );
+        }
+
+        if (
+            !\OmegaUp\Authorization::isCourseAdmin(
+                $r->identity,
+                $course
+            )
+        ) {
+            throw new \OmegaUp\Exceptions\ForbiddenAccessException();
+        }
+
+        $assignmentAlias = $r->ensureString(
+            'assignment_alias',
+            fn (string $alias) => \OmegaUp\Validators::alias($alias)
+        );
+
+        $assignment = \OmegaUp\DAO\Assignments::getByAliasAndCourse(
+            $assignmentAlias,
+            intval($course->course_id)
+        );
+        if (is_null($assignment)) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'assignmentNotFound'
+            );
+        }
+
+        if (
+            is_null(
+                \OmegaUp\DAO\ProblemsetProblems::getByPK(
+                    intval($assignment->problemset_id),
+                    $problem->problem_id
+                )
+            )
+        ) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'problemNotFoundInProblemset'
+            );
+        }
+
+        if (
+            $submission->problem_id !== $problem->problem_id ||
+            $submission->problemset_id !== $assignment->problemset_id
+        ) {
+            throw new \OmegaUp\Exceptions\ForbiddenAccessException(
+                'submissionNotAssociatedToProblem'
+            );
+        }
+
+        // Verificar si ya existe un feedback para esto.
+        $submissionFeedback = \OmegaUp\DAO\SubmissionFeedback::getBySubmission(
+            $submission
+        );
+        if (!is_null($submissionFeedback)) {
+            throw new \OmegaUp\Exceptions\DuplicatedEntryInDatabaseException(
+                'submissionFeedbackAlreadyExists'
+            );
+        }
+
+        $submissionAuthor = \OmegaUp\DAO\Identities::getByPK(
+            intval(
+                $submission->identity_id
+            )
+        );
+        if (is_null($submissionAuthor)) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'userNotExist'
+            );
+        }
+
+        \OmegaUp\DAO\SubmissionFeedback::create(
+            new \OmegaUp\DAO\VO\SubmissionFeedback([
+                'identity_id' => $r->identity->identity_id,
+                'submission_id' => $submission->submission_id,
+                'feedback' => $feedback,
+            ])
+        );
+
+        if ($submissionAuthor->user_id) {
```

Comment:
nit:

```suggestion
        if (!is_null($submissionAuthor->user_id)) {
```

es mejor hacer un check explícito contra null en vez de checar si es falsoso.